src/main/steps/applicant1/continue-with-your-application/content.ts

```diff
@@ -35,8 +35,32 @@ const en = ({ isDivorce, partner, isJointApplication, required }: CommonContent)
   },
 });
 
-// @TODO translations
-const cy = en;
+const cy = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```

Comment:
```suggestion
const cy: typeof en = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```

Reply:
Assigned type to the const as suggested by Lara